python/dask_cudf/dask_cudf/accessors.py

```diff
@@ -1,5 +1,53 @@
 # Copyright (c) 2021, NVIDIA CORPORATION.
 
+class StructMethods:
+    def __init__(self, d_series):
+        self.d_series = d_series
+    def field(self, key):
+        """
+        Extract children of the specified struct column
+        in the Series
+        Parameters
+        ----------
+        key: int or str
+            index/position or field name of the respective
+            struct column
+        Returns
+        -------
+        Series
+        Examples
+        --------
+        >>> s = cudf.Series([{'a': 1, 'b': 2}, {'a': 3, 'b': 4}])
+        >>> ds = dask_cudf.from_cudf(s, 2)
+        >>> ds.struct.field(0).compute()
+        0    1
+        1    3
+        dtype: int64
+        >>> ds.struct.field('a').compute()
+        0    1
+        1    3
+        dtype: int64
+        """
+        try:            
+            typ = self.d_series._meta.dtype.fields[key]
+            return self.d_series.map_partitions(
+                lambda s: s.struct.field(key),
+                meta=self.d_series._meta._constructor([], dtype=typ),
+            )
```

Comment:
Lets just set `typ` in the `try`/`except`, and move this return statement to the end (so we don't need to repeat it for both key-type scenarios)

Reply:
Thanks for the suggestion, Rick!  I like this much better